### ISSUE TITLE
Statistics section has mixed up values for "Who paid for this category?" and "What did they pay for?"

### ISSUE THREAD

**Issue by regdude** [julien-nc/cospend-nc#249]
Noticed that the "Who paid for this category?" and "What did they pay for?" values in the drop down menu are mixed up.
"Who paid for this category?" currently shows members, but "What did they pay for?" shows categories.

![image](https://github.com/julien-nc/cospend-nc/assets/74827193/5a722de0-af46-4c50-9ea3-df59b902a42c)

I believe the issue is in this file: https://github.com/julien-nc/cospend-nc/blob/main/src/components/statistics/Statistics.vue#L286

**Comment by julien-nc** [comment]
I'm not sure I understand what you mean.
* Who paid for this category? Select a category and what each member payed for it.
* What did they pay for? Select a member and see how much this member payed for each category

All seems fine to me.

**Comment by regdude** [comment]
You are right, the text confused me until I understood what the dropdown actually does.
Maybe you could consider changing the text so it is more obvious on what the filtering will do?

Something along the lines:
Who paid for this category? -> Show who paid for category $category
What did they pay for?  -> Show what did user pay for $user
